opendata-ogc-features/templates/opendata-ogc-features-config-cm.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if .Values.ogcFeatConfig }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  annotations:
+  labels: opendata-ogc-features
+  name: ogc-feat-config-cm
+data:
+  application.yaml: |
+    {{- .Values.ogcFeatConfig | nindent 4 }}
```

Comment:
missing a ` | toYaml ` ?

Reply:
seems to work like that need a change?

Reply:
from what I remember I had to cast the value as yaml in an other context, but if it works, i guess it's fine

Reply:
actually the ` | toYaml ` add a  ` |- ` not wanted :) so the good way is as I did